front/lib/api/extract.ts

```diff
@@ -154,3 +154,37 @@ export async function getExtractedEvents(
     };
   });
 }
+
+export async function deleteExtractedEvent(
+  auth: Authenticator,
+  id: string
+): Promise<boolean> {
+  const owner = auth.workspace();
+  if (!owner) {
+    throw "extracted_event_auth_error";
```

Comment:
We don't throw error types, we'd rather have a sentence for now

Reply:
(to ensure consistency with the rest of the files in `lib/api`)

Reply:
Yeah I wondered if that was okay or not, they are all caught in the API route handler and an `apiError` for each, but it's true if you call this function from elsewhere we would throw error code, I'll make the change.

Reply:
Yes as per below. If you catch your error then you should use a Result :+1: with a `Result` you'll be able to embed an error code and a message and you'll be able to embed that in the final API response as you see fit and the order will be restored :)